app/javascript/modules/listingDetails/ListingDetailsAdditionalInformation.tsx

```diff
@@ -18,12 +18,14 @@ export const ListingDetailsAdditionalInformation = ({
 }: ListingDetailsAdditionalInformationProps) => {
   const { unleashFlag: isSalesFcfsEnabled } = useFeatureFlag("FCFS", false)
 
+  listing.Realtor_Commission_Unit = "test"
+  listing.Realtor_Commission_Amount = null
```

Comment:
**nitpick (if-minor):** debugging clean up I think?

Reply:
Thanks! missed that!